package.json

```
@@ -48,17 +48,16 @@
     }
   },
   "files": [
-    "core/dist/**",
-    "infinite/dist/**",
-    "immutable/dist/**",
-    "mutation/dist/**",
-    "_internal/dist/**",
+    "core/dist/**/*.{js,d.ts,mjs}",
+    "infinite/dist/**/*.{js,d.ts,mjs}",
+    "immutable/dist/**/*.{js,d.ts,mjs}",
+    "mutation/dist/**/*.{js,d.ts,mjs}",
+    "_internal/dist/**/*.{js,d.ts,mjs}",
     "core/package.json",
     "infinite/package.json",
     "immutable/package.json",
     "mutation/package.json",
-    "_internal/package.json",
-    "package.json"
+    "_internal/package.json"
```

we need package.json for export map. there could be usage like `resolve('swr/package.json')`
```suggestion
    "_internal/package.json",
    "package.json"
```

---

#TIL
https://docs.npmjs.com/cli/v8/configuring-npm/package-json#files
> Certain files are always included, regardless of settings:
  package.json
README
LICENSE / LICENCE
The file in the "main" field